index.html

```diff
@@ -70,6 +70,10 @@
            ;;
     latest) url=https://api.github.com/repos/submariner-io/releases/releases/latest ;;
     devel|release*|feature*)
+        echo "https://github.com/submariner-io/subctl/releases/download/subctl-${VERSION}/subctl-${VERSION}-${os}-${architecture}.tar.xz"
+        return
+        ;;
+    feature-multi-active-gw|release-0.10|release-0.11|release-0.12)
```

Comment:
This needs to be above L72 otherwise, eg, `release-0.12` will match L72 and fail.